src/Fixer/LanguageConstruct/ClassKeywordFixer.php

```diff
@@ -69,7 +69,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
         for ($index = $tokens->count() - 1; $index >= 0; --$index) {
             $token = $tokens[$index];
 
-            if ($token->isGivenKind(T_CONSTANT_ENCAPSED_STRING)) {
+            if ($token->isGivenKind(T_CONSTANT_ENCAPSED_STRING) && str_contains($token->getContent(), '\\')) {
```

Comment:
what's wrong with replacing `"Redis"` with `Redis::class` if `$this->exists("Redis")` returns true ?

Reply:
What if this is something we want to do in the seeder(example below) or is not related to Redis::class at all?

```
Technology::factory()->createMany([
            ["name" => "Laravel"],
            ["name" => "Symfony"],
            ["name" => "CakePHP"],
            ["name" => "PHP"],
            ["name" => "Livewire"],
            ["name" => "Inertia"],
            ["name" => "Vue"],
            ["name" => "Javascript"],
            ["name" => "Redis"],
        ]);
```

Reply:
And what if we want to do on other case is related to `Redis::class` ?
This automated rule will never know. we could have `too many` or `not enough` cases covered.

Rule is risky and very big warning for users to use it.
I prefer rule to automate as much as possible and user may decide to not commit everything.